test/server/cards/02_SHD/leaders/FinnThisIsARescue.spec.ts

```diff
@@ -99,27 +86,26 @@ describe('Finn, This is a Rescue', function () {
                     }
                 };
 
+                // TODO - should we remove this comment now that ownership is handled in setup?
```

Comment:
I think we can remove the comment yeah, it's unclear who it's even for since the tests are already done